src/main/resources/jenkins/branch/OverrideIndexTriggersJobProperty/help.html

```diff
@@ -0,0 +1,11 @@
+<div>
```

Comment:
You might want to make this `help-enableTriggers.html`. Not sure which looks better in practice.